src/main/scala/no/ndla/articleapi/model/search/SearchableArticle.scala

```diff
@@ -25,5 +25,6 @@ case class SearchableArticle(
     authors: Seq[String],
     articleType: String,
     defaultTitle: Option[String],
-    grepCodes: Seq[String]
+    grepCodes: Seq[String],
+    conceptIds: Seq[Long]
```

Comment:
Er ikkje sikkert vi treng å ha dette i SearchabeArticle. Men då bør du ha med filtrering på artikler basert på dette også. Samt at mapping for indeksen også oppdateres. Og så må det samme gjøres i draft-api sjølvsagt.

Reply:
Er det heller ikke ønsket at det skal være en del av ArticleSummary?

Reply:
ArticleSummaryV2 brukes ved presentering av søkeresultat. Ser ikkje heilt behovet for å ha det med der heller. Det er først ved visning av artikkelen det gir meining å presentere lista med forklaringer.